### DISCUSSION TITLE
headerClassname does not work

### DISCUSSION THREAD

**Discussion by tqdo** [TanStack/table#2726] [General]
I am trying to make the first column in my React Table sticky, however I could not quite get it to work. The first column may still disappear when I scroll horizontally.

What I am doing is in `columns`

````
{
                Header: "First Name",
                accessor: "firstName",
                headerClassname: 'sticky_col'
}
````

CSS

````
.sticky_col {
    position: -webkit-sticky;
    position: sticky;
    left:0;
    background-color: white;
}
````
It seems like `headerClassname` has no effect as I have tried other styling, none of them has any effects. Thanks

**Comment by carlbrn** [comment]
You can see in this example how to add a classname or a style for the header and the cell:
[codesandbox example](https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/data-driven-classes-and-styles)

Basically it's like this:

```javascript
<th {...column.getHeaderProps([{
  className: cell.column.className,
  style: cell.column.style,
}])}>
```
